tasks/RevolutionTask.cpp

```diff
@@ -1,21 +1,33 @@
 /* Generated from orogen/lib/orogen/templates/tasks/Task.cpp */
 
 #include "RevolutionTask.hpp"
+#include <algorithm>
+#include <base-logging/Logging.hpp>
 
 using namespace std;
 using namespace base;
 using namespace deep_trekker;
 using namespace iodrivers_base;
 
 RevolutionTask::RevolutionTask(std::string const& name)
-    : RevolutionTaskBase(name)
+    : RevolutionTaskBase(name), m_camera_head_tilt_position(base::unknown<double>())
 {
 }
 
 RevolutionTask::~RevolutionTask()
 {
 }
 
+void tryWriteIgnoringExceptions(function<void()> write_func)
```

Comment:
`tryParseAndWrite`

This can be (and has been by me) interpreted as "the write throws an exception", but it's actually the get.